examples/Manifest.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.3"
+julia_version = "1.7.0"
```

Comment:
@charleskawczynski I used julia 1.7.0 which results in this change. Could I just change it back to 1.7.3, or do I need to change something else as well?

Reply:
I think it's probably fine, but you should upgrade!

Reply:
ok I will! :)